src/RAI.jl

```diff
@@ -46,10 +46,10 @@ export
     list_engines
 
 export
-    delete_model,
+    delete_models,
```

Comment:
If you are making a breaking change like this, we'll have to bump the version number in the Project.toml to be a breaking change.

It would probably be better to leave the old names, and mark them as deprecated, and print a deprecation warning, like we did in https://github.com/RelationalAI/rai-sdk-julia/pull/61, for create_database:
https://github.com/RelationalAI/rai-sdk-julia/blob/d2da71d3dc49ee14959941d9dad4977cbe64cc1b/src/api.jl#L398-L402

In this case, it'd be even easier, you just keep the old method, print the deprecation warning, and forward to the new method.

Then we can delete these when we bump the major version number in a future release. Does that make sense?

Reply:
Or if you'd rather not do this now, you can just bump the major version number now, and make this a breaking change, but I would check with @torkins and @bradlo about whether we have the appetite for a breaking release here, or if it'd be better to do this via deprecation warning + new function. I'd personally rather go the non-breaking approach.

Reply:
technically, we are only required to bump the version when we release a version. as for leaving deprecated entries, its really a judgement call .. given the number of breaking changes we are and will be making, there is real overhead to doing this, otoh if its a widely used entry point, then it may be worth it. and if its obscure, it may not. the only hard requirement is that all feature adds and breaking changes are noted in the CHANGEME.md file.